relay-event-normalization/src/normalize/span/description/sql/mod.rs

```diff
@@ -211,8 +211,8 @@ mod tests {
 
     scrub_sql_test!(
         various_parameterized_ins_percentage,
-        "SELECT count() FROM table1 WHERE id IN (%s, %s) AND id IN (%s, %s, %s)",
-        "SELECT count() FROM table1 WHERE id IN (%s) AND id IN (%s)"
```

Comment:
I think this raises an edge case how queries like this one where we have the same condition twice joined by a `AND` operator.

Likely an uncommon use case, but the first list may be a subset of the second so it may be valid to transform it into just `id IN (%s)`.

Reply:
I think it's OK to pretend that they are the same for grouping purposes, with the same reasoning as for the ` OR ` clause: If the left operand (`id`) and the operator ` IN ` repeat multiple times, it is likely a dynamically generated query that we can simplify.